packages/client/composables/useClicks.ts

```diff
@@ -16,10 +16,10 @@ export function createClicksContextBase(
   return {
     get current() {
       // Here we haven't know clicksTotal yet.
-      return Math.max(+current.value, clicksStart)
```

Comment:
I am not very sure if this is OK. Because `current` can be accessed at any time, but `this.total` is only available after the whole slide has been mounted.

Reply:
I think accessing it in `current` is just like anywhere else the existing `total` only guarantees the correct behavior of the final application.